### DISCUSSION TITLE
What to do with logos with underscores?

### DISCUSSION THREAD

**Discussion by vicw0ng-hk** [badges/shields#6571] [Q&A] [answered]
For example, logo for Pop!_OS. Simple Icons names it [Pop!_OS](https://simpleicons.org/?q=pop!_os), but this does not work:
```
![Pop!_OS](https://img.shields.io/badge/-Pop%21__OS-fff?&logo=pop%21_os&logoColor=48B9C7)
```
![Pop!_OS](https://img.shields.io/badge/-Pop%21__OS-fff?&logo=pop%21_os&logoColor=48B9C7)

Neither does
```
![Pop!_OS](https://img.shields.io/badge/-Pop%21__OS-fff?&logo=pop%21__os&logoColor=48B9C7)
```
![Pop!_OS](https://img.shields.io/badge/-Pop%21__OS-fff?&logo=pop%21__os&logoColor=48B9C7)

What is the right way?

**Comment by calebcartwright** [comment, marked as answer]
The Simple Icons slug is currently `pop_os`. So you'd do:

https://img.shields.io/badge/-Pop%21__OS-fff?&logo=pop_os&logoColor=48B9C7
![](https://img.shields.io/badge/-Pop%21__OS-fff?&logo=pop_os&logoColor=48B9C7)

Worth noting that they did just recently change the slug to `popos` so when we pull in that next update of the simple-icons package the slug will change accordingly
